Ginger/Ginger/UserControlsLib/UCSendEMailConfig/UCSendEMailConfigView.xaml

```diff
@@ -0,0 +1,511 @@
+﻿<UserControl x:Class="Ginger.UserControlsLib.UCSendEMailConfig.UCSendEMailConfigView"
```

Comment:
Better to rename UC name to 'UCEmailConfigView' as same control will be used for send and read emails.

Reply:
Renamed the user control